.test-infra/jenkins/job_LoadTests_Combine_Flink_Go.groovy

```diff
@@ -133,5 +133,5 @@ CronJobBuilder.cronJob('beam_LoadTests_Go_Combine_Flink_Batch', 'H 8 * * *', thi
     influx_hostname: InfluxDBCredentialsHelper.InfluxDBHostUrl,
   ]
   // TODO(BEAM-12898): Re-enable this test once fixed.
-  // loadTestJob(delegate, CommonTestProperties.TriggeringContext.POST_COMMIT, 'batch')
+  //loadTestJob(delegate, CommonTestProperties.TriggeringContext.POST_COMMIT, 'batch')
```

Comment:
We can re-enable these. It's fine if they're failing as long as we're not leaking VMs (which I believe was fixed in #15547).